src/components/pages/ConfirmationPage.jsx

```diff
@@ -154,14 +162,15 @@ function ConfirmationPage() {
                 onChange={(e) => handleInputChange('message', e.target.value)}
               ></textarea>
             </div>
-
             <button className="form-field" type="submit">
               { t('submitText') }
             </button>
           </div>*/}
```

Comment:
I know this wasn't your work but can you remove this commented out div?

Reply:
on it!!